.wp-env.json

```diff
@@ -1,7 +1,7 @@
 {
 	"core": "WordPress/WordPress",
 	"plugins": [ "." ],
-	"themes": [ "WordPress/theme-experiments/tt1-blocks#tt1-blocks@0.4.7" ],
+	"themes": [ "WordPress/theme-experiments/emptytheme#emptytheme#8f5e564b366db75e6067244ef54b91d06d4f8d8c" ],
```

Comment:
We could probably use a new tag from that repo.

Reply:
Creating a tag sounds like a good idea, probably after porting changes that are needed to the theme.